.travis.yml

```diff
@@ -32,6 +32,19 @@ matrix:
       before_script:
         - . ./ci/travis/ci.sh build
 
+    - os: linux
+      env:
+        - PYTHON=3.6
+        - PYTHONWARNINGS=ignore
+        - RAY_DEFAULT_BUILD=1
+        - RAY_CYTHON_EXAMPLES=1
+        - RAY_USE_RANDOM_PORTS=1
+        - RAY_ENABLE_MULTI_TENANCY=1
```

Comment:
Can you put this in the first line so that people can easily know what this CI is about?